internal/controller/spannerautoscaler_controller.go

```diff
@@ -485,7 +500,7 @@ func (r *SpannerAutoscalerReconciler) needUpdateProcessingUnits(log logr.Logger,
 		log.Info("no need to scale", "currentPU", currentProcessingUnits, "currentCPU", sa.Status.CurrentHighPriorityCPUUtilization)
 		return false
 
-	case desiredProcessingUnits > currentProcessingUnits && r.clock.Now().Before(sa.Status.LastScaleTime.Time.Add(10*time.Second)):
+	case desiredProcessingUnits > currentProcessingUnits && r.clock.Now().Before(sa.Status.LastScaleTime.Time.Add(getOrConvertTimeDuration(sa.Spec.ScaleConfig.ScaleupInterval, r.scaleUpInterval))):
```

Comment:
@rustycl0ck 
I think it's better to set default scaleupInterval as 10 seconds because default interval is 10 seconds before this PR.
How do you think?

Reply:
Setting default `scaleUpInterval` as less than 60 doesn't work anything because reconcile interval is 60 seconds. So I set the default value as 60 seconds.